examples/README.md

```diff
@@ -34,6 +34,10 @@ To quickly locate the examples of your interest, search for the tagged keywords
 - <a name='ngnn'></a> Song et al. Network In Graph Neural Network. [Paper link](https://arxiv.org/abs/2111.11638).
     - Example code: [PyTorch](../examples/pytorch/ogb/ngnn)
     - Tags: model-agnostic methodology, link prediction, open graph benchmark.
+- <a name='bipointnet'></a>Qin et al. BiPointNet: Binary Neural Network for Point Clouds. [Paper link](https://openreview.net/forum?id=9QLRCVysdlO)
+    - Example code: [PyTorch](../examples/pytorch/pointcloud/pointnet/bipointnet_cls.py)
+    - Tags: model binarization, model compression and acceleration.
```

Comment:
```suggestion
    - Example code: [PyTorch](../examples/pytorch/pointcloud/bipointnet)
    - Tags: point cloud classification, network binarization
```
I suggest putting this example in a separate folder. Besides, the folder should contain a readme file to describe its purpose, usage, and results (considering the focus of BiPointNet on network binarization, results on performance and efficiency are both required), etc. But sure you can add a pointer to it from the readme of [PointNet](https://github.com/dmlc/dgl/blob/master/examples/pytorch/pointcloud/pointnet/README.md).

Reply:
Hello reviewer, does the current version meet the requirements？